src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/hmi_notifications_test.cc

```diff
@@ -1877,8 +1916,8 @@ TEST_F(HMICommandsNotificationsTest, OnDriverDistractionNotificationValidApp) {
       hmi_apis::Common_DriverDistractionState::DD_ON;
   MessageSharedPtr message = CreateMessage();
   (*message)[am::strings::msg_params][am::mobile_notification::state] = state;
-  utils::SharedPtr<Command> command =
-      CreateCommand<hmi::OnDriverDistractionNotification>(message);
+  utils::SharedPtr<Command> command = CreateCommand<
+      sdl_rpc_plugin::commands::hmi::OnDriverDistractionNotification>(message);
```

Comment:
@ValeriiMalkov May be it is worth to think about adding `using namespace sdl_rpc_plugin::commands` to the begining of file?